example/setupapi.dart

```diff
@@ -17,9 +17,9 @@ void main() {
     final deviceInfoSetPtr =
         SetupDiGetClassDevs(deviceGuid, nullptr, NULL, DIGCF_PRESENT);
     try {
-      final deviceHandles = iterateDeviceHandle(deviceInfoSetPtr, deviceGuid);
-      for (final handle in deviceHandles) {
-        print('net device ${handle.toHexString(32)}');
+      final deviceHandles = deviceIntancesByClass(deviceInfoSetPtr, deviceGuid);
```

Comment:
```suggestion
      final deviceHandles = deviceInstancesByClass(deviceInfoSetPtr, deviceGuid);
```

typo in this one, I think?